test-data/ert/heat_equation/generate_files.py

```diff
@@ -71,6 +72,19 @@ def make_observations(
     return d
 
 
+def generate_priors():
```

Comment:
Hm, it would be nice to have a bit more docs on this maybe. Does it create 10 fields each of (nx, nx) of size?

Reply:
Good catch. The reshape is not needed. I've also added a docstring.